app/lib/trs/params/professional_status.rb

```diff
@@ -57,7 +57,7 @@ def params
         @params ||= {
           "routeTypeId" => route_type_id,
           "status" => status,
-          "awardedDate" => trainee.awarded_at&.to_date&.iso8601,
+          "awardedDate" => trainee.outcome_date&.to_date&.iso8601,
```

Comment:
discussed here https://ukgovernmentdfe.slack.com/archives/C07UWPWFCJ2/p1746615841696779 (or see screenshot in earlier comment)